connection-api/src/main/java/org/terracotta/connection/ConnectionPropertyNames.java

```diff
@@ -35,4 +35,6 @@
   String CONNECTION_DISABLE_RECONNECT = "connection.noreconnect";
   /*  request a specific connection type */
   String CONNECTION_TYPE = "connection.type";
+  /*  request a specific connection scheme */
+  String CONNECTION_SCHEME = "connection.scheme";
```

Comment:
Probably makes more sense as an argument in connect.  If this stays as a property, there is overlap here with CONNECTION_TYPE.  Either make this an argument in connect or merge your additional definitions into CONNECTION_TYPE